docs/release_notes/0.87.0.md

```diff
@@ -2,5 +2,10 @@
 
 ## Features
 
-- Added validation check for zones before creating cluster (#4837)
+- Add validation check for zones before creating cluster (#4837)
+
+## Bug Fixes
+
+- Fix missing stack name in case only the stack is provided. (#4894)
```

Comment:
```suggestion
- Fix missing stackname when updating addons (#4894)
```

Reply:
🙃 
![Screenshot 2022-03-04 at 11 39 07](https://user-images.githubusercontent.com/10995114/156757024-d8ad2a25-97c7-4b0a-9fd8-b9d8ec7e12f4.png)

Reply:
It made sense in the context of the PR.

Reply:
Also, you always say to tick boxes. ;)

Reply:
Ticking the box implies that you performed the action. We should make sure the title of the PR is good enough instead of tweaking release notes

Reply:
This is why I always clear the PR description. ;) The boxes are meaningless.

Reply:
I am talking about the PR title here. 

> The boxes are meaningless.

Let's find some meaningful ones then.

Reply:
cc @gargath

Reply:
> Let's find some meaningful ones then.

Any kind of "box" is meaningless. When I review a PR I will look at the code, I don't care about the boxes at all. When I create a PR I know what I did or did not do and most of the time the relevant issue will have all the descriptions. However, I do agree that a structural template is a good idea. Something like:

```
# Description
# Changes
# Affected behaviour if any
# Linked Issue
```

That way when anyone outside of the team or inside the team creates a PR they will know what to expect / to do / how to give enough context.

Ticking boxes will not give anyone any information about the content of a PR. :)